test/consts.test.ts

```diff
@@ -6,12 +6,9 @@ describe('consts', () => {
         it('REGEX works as expected', () => {
             expect(USERNAME.REGEX.test('anonymous')).toBe(true);
             expect(USERNAME.REGEX.test('---')).toBe(true);
-            expect(USERNAME.REGEX.test('john.doe')).toBe(true);
```

Comment:
instead of removing tests, you should just change the assertions

Reply:
Agree. Will fix.

Reply:
Fixed.